KeeAgent/ExtensionMethods.cs

```diff
@@ -295,5 +295,19 @@ public static void ReplaceWithGlobalConfirmMessage(this Control control)
         Size = control.Size,
       });
     }
+
+      /// <summary>
+      /// Expand environment variables in a filename. Also expandes ~/ to %HOME% environment variable.
+      /// </summary>
+      /// <param name="path"></param>
+    public static String ExpandEnvironmentVariables(String filename)

```

Comment:
To make this an extension method, write

    public static String ExpandEnvironmentVariables(this String filename)